### PR TITLE
scripts/build_utils.sh: install gnupg before ppa packages

### PR DESCRIPTION
**Pull request by tchaikov**
so gnupg can be installed for checking the ppa repo's signatures.

Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

File: scripts/build_utils.sh [hunk 1]
```diff
@@ -655,6 +655,13 @@ setup_pbuilder_for_new_gcc() {
     local version=$1
     shift
 
+    cat >> $hookdir/D05install-gcc-7 <<EOF
+echo "deb [lang=none] http://ports.ubuntu.com/ubuntu-ports $DIST-updates main" >> \
```
Comment (dmick) [inline scripts/build_utils.sh:659]: 1) I think this should be http://archive.ubuntu.com/ubuntu/dists/$DIST-updates, no?
2) it ought to go on the end of /etc/apt/sources.list, as it's part of the ubuntu sources

Comment (dmick) [reply scripts/build_utils.sh:659]: (at least for x86_64; it's possible arm64 is different)

Comment (tchaikov) [reply scripts/build_utils.sh]: @dmick fixed and repushed.

**Comment by tchaikov** [comment]
folks, i just deployed the change in this PR to address the FTBFS of nfs-ganesha.

**Review by jdurgin — APPROVED** [review]

**Pull request merged**